src/http/server.ts

```diff
@@ -473,7 +487,7 @@ export class Server {
       options.hostname = this.hostname;
     }
     if (!options.port) {
-      options.port = 1447;
+      options.port = this.port;
     }
     this.hostname = options.hostname;
```

Comment:
need `this.port = options.port` here

Reply:
[Done](https://github.com/drashland/deno-drash/pull/198/commits/be96d393acc92dad7cda2f7489606a40ac92b6c4).